### PR TITLE
New commands: true and false

### PR DESCRIPTION
**Pull request by nfischer**
These are pretty similar, so I wrapped these into one PR.

`true` does nothing and returns status `0`. `false` does nothing and returns a non-zero status (for me on Ubuntu, it returns `1`).

I added unit tests, but I couldn't figure out how to check the actual return code (I only check the `stdout`). @dthree if you could direct me on how to check the return status, that would be good (if this sort of thing is even possible).

Also, it would be good if return status of the last command was saved somewhere globally (or inside `vorpal`, like the aliases are). This would let us represent `$?` (the return value of the last command).

I tried adding these as separate packages, but I'm not sure if I specified the correct dependencies. As far as I can tell, these should require minimal dependencies, so please check to see if I did this correctly, and I can revise accordingly.

### CODE REVIEW

File: src/help/true.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module.exports = `
+Usage: true [OPTION]
+Do nothing, successfully
+
+This command simply exits with status zero (success).
+
+      --help   display this help and exit
+
+Report export bugs to <https://github.com/dthree/cash>
```
Comment (dthree) [inline src/help/true.js]: > Report true bugs to <https://github.com/dthree/cash>

File: src/help/false.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = `
+Usage: false [OPTION]
+Do nothing, unsuccessfully
+
+This command simply exits with status 1 (failure).
+
+      --help   display this help and exit
+
+Report export bugs to <https://github.com/dthree/cash>
```
Comment (dthree) [inline src/help/false.js]: > Report false bugs to <https://github.com/dthree/cash>

File: src/help/true.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module.exports = `
+Usage: true [OPTION]
+Do nothing, successfully
+
+This command simply exits with status zero (success).
```
Comment (dthree) [inline src/help/true.js]: Can you turn `zero` into `0` for consistency with the `false` help?

File: commands.json [hunk 4]
```diff
@@ -71,6 +73,12 @@
         "./dist/util/interfacer.js"
       ]
     },
+    "false": {
+      "dependencies": [],
```
Comment (dthree) [inline commands.json]: You did fine here - no deps needed! :sunglasses:

**Comment by dthree** [comment]
This all looks great - there's just 3 small points in the help pages. Fix that and I can merge.

---

> Also, it would be good if return status of the last command was saved somewhere globally (or inside vorpal, like the aliases are). This would let us represent $? (the return value of the last command)

We should create a new issue to add this in as a new enhancement.

File: src/help/true.js (hunk 1 above)
Comment (nfischer) [inline src/help/true.js]: Whoops! Good catch. I'll fix that

File: src/help/true.js (hunk 3 above)
Comment (nfischer) [inline src/help/true.js]: Will do

Comment (dthree) [inline src/help/true.js]: :+1:

**Comment by nfischer** [comment]
> We should create a new issue to add this in as a new enhancement.

SGTM

File: src/help/true.js (hunk 3 above)
Comment (nfischer) [inline src/help/true.js]: Done

File: src/help/false.js (hunk 2 above)
Comment (nfischer) [inline src/help/false.js]: Done

File: src/help/true.js (hunk 1 above)
Comment (nfischer) [inline src/help/true.js]: Done

**Comment by nfischer** [comment]
Should be good now

**Comment by dthree** [comment]
Awesome thanks! :+1:

**Pull request merged**